src/Umbraco.Core/Models/Mapping/MemberTabsAndPropertiesMapper.cs

```diff
@@ -26,8 +28,13 @@ public class MemberTabsAndPropertiesMapper : TabsAndPropertiesMapper<IMember>
     private readonly IMemberService _memberService;
     private readonly IMemberGroupService _memberGroupService;
     private readonly MemberPasswordConfigurationSettings _memberPasswordConfiguration;
-    private readonly PropertyEditorCollection _propertyEditorCollection;
+    private readonly ITwoFactorLoginService _twoFactorLoginService;
```

Comment:
Might be out of scope entirely, but I really do not like services in mappers, in theory, they should just be mapping 1:1, and if services are needed, we should use a factory instead 🤔 
This is of course major refactoring so will take a bit of time, so might be a great topic for NTT to discuss how we want to move forward👍

Reply:
Yeah, let's bring that up on Monday 😁